lib/src/components/NxTab/NxTab.scss

```diff
@@ -6,16 +6,32 @@
  */
  @import '../../scss-shared/nx-colors';
  @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
 
 .nx-tab {
-  border-bottom: 4px solid transparent;
+  border-bottom: 2px solid transparent;
   cursor: pointer;
   display: inline-block;
   list-style: none;
-  padding: $nx-spacing-xs $nx-spacing-l;
+  margin-right: $nx-spacing-md;
+  // subtract border thickness for consistent spacing
+  padding: $nx-spacing-md 0 ($nx-spacing-md - 2px) 0;
```

Comment:
Not 24px on the sides?

Reply:
> Not 24px on the sides?

After clarification from Derrek: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/44a4cdae7cb8121108a56d03c97a62433c7e8074